pkgs/development/libraries/libagar/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, pkgconfig, libtool, perl, bsdbuild, gettext, mandoc
+, libpng, libjpeg, xlibsWrapper, libXinerama, freetype, SDL, mesa
+, libsndfile, portaudio, mysql, fontconfig
+}:
+
+let
+  srcs = import ./srcs.nix { inherit fetchurl; };
+in
+stdenv.mkDerivation rec {
+  name = "libagar-${version}";
+  version = srcs.version;
+
+  src = srcs.src;
+
+  hydraPlatforms = [];
+
+  preConfigure = ''
+    substituteInPlace configure.in \
+      --replace '_BSD_SOURCE' '_DEFAULT_SOURCE'
+    cat configure.in | ${bsdbuild}/bin/mkconfigure > configure
+  '';
+
+  configureFlags = [
+    "--with-libtool=${libtool}/bin/libtool"
+    "--enable-nls=yes"
+    "--with-gettext=${gettext}"
+    "--with-jpeg=${libjpeg.dev}"
+    "--with-gl=${mesa}"
+    "--with-mysql=yes"
+    "--with-manpages=yes"
+  ];
+
+  outputs = [ "out" "devdoc" ];
+
+  nativeBuildInputs = [ pkgconfig libtool gettext ];
+  buildInputs = [
+    bsdbuild perl xlibsWrapper libXinerama SDL mesa  mysql.client mandoc
+    freetype.dev libpng libjpeg.dev fontconfig portaudio libsndfile
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A cross-platform GUI toolkit.";
```

Comment:
sorry keep forgetting about those pesky dots... Someone really should submit a nixfmt to the review process ;)